composer-lagoon.json

```diff
@@ -62,6 +62,16 @@
         "patchLevel": {
             "drupal/core": "-p2"
         },
+        "enable-patching": true,
+        "patches": {
+            "drupal/events_log_track": {
```

Comment:
We shouldn't need to patch in two locations - we should only patch modules/core in the composer.json - have at look at #213 - it may give a clue why these might not be applying properly

Reply:
sorry - I was half right - we don't need the patches in the composer-lagoon, but we do need to enable patching (in order for the patches inherited from dependencies (i.e. the govcms8 composer.json) to work